libvast/include/vast/uuid.hpp

```diff
@@ -38,7 +38,7 @@ class uuid : detail::totally_ordered<uuid> {
   using size_type = size_t;
 
   static uuid random();
-  static uuid nil();
+  static uuid null();
```

Comment:
Technically speaking an unrelated change, but I think this is fine. I like this better.